plugins/automation/db/migrate/20250319232839_add_automation_stats.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+class AddAutomationStats < ActiveRecord::Migration[7.2]
+  def change
+    create_table :discourse_automation_stats do |t|
+      t.bigint :automation_id, null: false
+      t.date :date, null: false
+      t.datetime :last_run_at, null: false
+      t.float :total_time, null: false
+      t.float :average_run_time, null: false
+      t.float :min_run_time, null: false
+      t.float :max_run_time, null: false
```

Comment:
Was considering milliseconds to get these down to integer, but I think the convenience of just passing seconds around is better. 👍